scalalib/src/GenIdeaModule.scala

```diff
@@ -17,6 +17,11 @@ trait GenIdeaModule extends Module {
    */
   def skipIdea: Boolean = false
 
+  /**
+   * Map containing a map of source folder names to package prefixes for Idea project file generation.
+   */
+  def packagePrefix: Map[String, String] = Map.empty[String, String]
```

Comment:
Just using folder names isn't enough. Consider two `src` folders in different parent directories. Best is to use a `os.Path` as key.